karma.conf.js

```diff
@@ -0,0 +1,70 @@
+// Karma configuration
+// Generated on Tue Mar 14 2017 06:19:19 GMT+0530 (IST)
+
+module.exports = function(config) {
+    config.set({
+
+        // base path that will be used to resolve all patterns (eg. files, exclude)
+        basePath: '',
```

Comment:
We can set this to parent folder from which all other files will take the `path`.
Here, it can be taken as `../..` also specified in karma.config docs.
http://karma-runner.github.io/1.0/config/configuration-file.html